assets/admin/tour/helper.js

```diff
@@ -0,0 +1,87 @@
+export const HIGHLIGHT_CLASS = 'sensei-tour-highlight';
+
+export function performStepAction( index, steps ) {
```

Comment:
Should we also add a JSDoc here?

Reply:
Thanks! Added here https://github.com/Automattic/sensei/pull/7535/commits/d7a2440bf5351a473f97328628e4d88120ba1043